docs/download.md

```diff
@@ -12,8 +12,8 @@ bodyclass: download-page
 		<th>Description</th>
 	</tr>
 	<tr>
-		<td><a href="https://leafletjs-cdn.s3.amazonaws.com/content/leaflet/v1.9.1/leaflet.zip">Leaflet 1.9.1</a></td>
-		<td>Stable version, released on September 23, 2022.</td>
+		<td><a href="https://leafletjs-cdn.s3.amazonaws.com/content/leaflet/v1.9.2/leaflet.zip">Leaflet 1.9.2</a></td>
+		<td>Stable version, released on October 4, 2022.</td>
 	</tr>
 	<tr>
 		<td><a href="https://leafletjs-cdn.s3.amazonaws.com/content/leaflet/v1.8.0/leaflet.zip">Leaflet 1.8.0</a></td>
```

Comment:
This still lists 1.8.0, not 1.9.1. Perhaps this was intentional? @mourner

Reply:
Yeah, let's keep it for now.